include/libsemigroups/paths.hpp

```diff
@@ -793,6 +438,7 @@ namespace libsemigroups {
   //! Returns the number of paths starting at a given node with length in a
   //! given range.
   //!
+  //! \param wg the WordGraph
```

Comment:
```suggestion
  //! \param wg the WordGraph.
```
(and the same for the params below)